regular/coc.html

```diff
@@ -0,0 +1,160 @@
+    <li>ableism</li>
+    <li>racism</li>
+    <li>COVID-19</li>
+    <li>references to suicide or self harm</li>
+</ul>
+</p>
+<p>
+    Some topics that people might not want to deal with, like food or politics, have their own channels, and efforts should be made to keep those topics to those channels.  
+</p>
+<p>
+    It is helpful if CWs are fairly descriptive, as some people will want to know about, say, policies that might affect them personally, but not be able to deal with seeing random hate against them. for example, one might have the following CWs for transphobia
+</p>
+<p>
+    [reference to transphobic policy, funny]<br />
+    <i>a gallows humour tweet about a new policy</i>
+</p>
+<p>
+    [transphobic policy, new development]<br />
+    <i>a new update about said transphobic policy</i>
+</p>
+<p>
+    [transphobia, bad take]<br />
+    <i>a link to a bad tweet</i>
+</p>
+<p>
+    When talking about a specific event, person or group, you may want to include them in the CW, like so:
+</p>
+<p>
+    [rms, ecofascism, bad take]
+</p>
+<p>
+    That said, if simply reposting a bad take, please consider whether it’s necessary - bear in mind that repeating lies (even by condemning them) makes it more likely that people will believe them, and the people affected by the take are quite likely to have already seen it. Also, there are certain topics that may be very traumatic to some people even if accompanied by a content warning, so think carefully whether it should be posted at all.
+</p>
+<p>
+    Some people choose to put ranting/talking about being upset behind content warnings. While this might be useful for some people, it is not a requirement, nor should be apologised for if one forgets to do it.
+</p>
+<h3>IRC interlopers</h3>
+<p>
+    Sometimes people join our chat that may not be familiar with us, or may not be students at the University of York altogether. This can happen because anyone with an @york.ac.uk email can join the Slack, but also because anyone can join the IRC and our website is somehow one of the top Google results for “computer science irc”. Be nice and welcoming to them.
+</p>
+<p>
+    Sometimes they may ask us to do their homework or ask an oddly specific CS question that’s clearly an assignment. Feel free to point them in the right direction, but consider not doing their homework for them. Not only can it be a violation of York’s academic misconduct policy or their institution’s honour code, but part of learning CS is learning how to solve problems, which someone solving it for you doesn’t teach. (Incidentally, if you’ve stumbled upon this page via Google, hi! Feel free to join and chat, but please don’t ask us to do your homework.)
+</p>
+<h3>Profiles</h3>
+<p>
+    Some people put their pronouns under “what I do” on Slack, or in their nickname on Discord. This is useful for others, but not required. Consider occasionally checking people’s profiles to see if their pronouns have changed.
+</p>
```

Comment:
nit: something something newlines at ends of files :^)

also, the `<p>` styling in this file isn't super consistent (some lines have `<p>` on the same line as the text, some split the tags and contents over multiple lines), but i don't care thaaaat much